src/controllers/roles.js

```diff
@@ -7,6 +7,13 @@ const getRoles = (req, res) => {
 
 // TODO: Create a new Role
 
+const getName = (req, res) => {
```

Comment:
this controller is ok but how do we use it? What is the path to access this resource? 🤔